### DISCUSSION TITLE
Rust Dedicated Server: (Ubuntu) Unable to get into RCON

### DISCUSSION THREAD

**Discussion by Consumedgrub2** [GameServerManagers/LinuxGSM#3927] [Q&A] [answered]
I'm having an issue where the rust config is properly set and I define my own rcon password and leave everything else like port and IP default, however when I go to use [RustAdmin](https://www.rustadmin.com/), I put in all my info and it fails to connect. And yes rcon web is set 1. I have spent 8+ hours and have found no documentation on this issue of how I'm unable to connect via rcon to give myself admin. I am able to connect normally though the game client and see the correct server description and title, its just when I launch the linux rust server, I get an output that tells me the command line configuration, showing me things like -server.hostname and stuff like that and the rcon password is present, (It also claims that rcon.password is not a command which tells me it sees things like a command? what?) however I just can't connect and it seems to be a problem with lgsm and not rustadmin or the linux configuration since I've checked to make sure all dependencies are installed.

**Comment by ElZeRo69** [comment]
are you using the right port (RCON Port)? default is 28016 and it has to be forwarded.

**Reply by Consumedgrub2** [reply]
All ports are fowarded on my vps and it is using the default rcon port, I've tried hosting the rust servers without using lgsm, and just a vanilla plain one downloaded from steam, and I was able to rcon into that, however I'm new to linux so I have no clue on how to start the rust server with arguments on the linux command line (like -server.maxplayers argument) (I'm new to bash). So instead, I thought having a config file that came with lgsm would be nice, so I opted for lgsm, however I was able to get everything to work EXCEPT for rcon-ing into the server. Either something is wrong with my lgsm, and instead I would opt for the plain dedicated rust server without lgsm (But I would have to figure out how to make a bash or shell script to start the server with the parameters I want) or figure out the RCON issue with lgsm.

**Comment by Consumedgrub2** [comment, marked as answer]
Never Mind, I figured it out, it turns out what I was doing before with the plain rust server without using lgsm was the right way to do it, since the command line issues I was having to get my config working was wrong, and so I just stopped using a bash/shell script and instead just typed in parameters on the command line when starting the server and all the config as well as the rcon worked. There was genuinely an issue with the rcon on the lgsm server though.

For anyone wondering the same thing and are new to linux, this is the exact command I used to get the rust server to launch with the parameters:
`./RustDedicated +server.hostname "theserverhostname" +server.port 28015 +server.level "Procedural Map" +server.seed 123 +server.maxplayers 150 +server.worldsize 3500 +server.saveinterval 1000 +rcon.password "change this, and leave the quotation marks on" +rcon.port 28016 +server.description "A server." +rcon.web 1 +server.identity "the name of the file the server will generate"`
